lib/jblond/Diff/Renderer/SubRenderer.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+declare(strict_types=1);
+
+namespace jblond\Diff\Renderer;
+
+/**
+ * Sub rendering class interface for PHP DiffLib.
+ *
+ * PHP version 7.2 or greater
+ *
+ * @package     jblond\Diff\Renderer\Html
+ * @author      Ferry Cools <info@DigiLive.nl>
+ * @copyright   (c) 2020 Ferry Cools
+ * @license     New BSD License http://www.opensource.org/licenses/bsd-license.php
+ * @version     2.0.0
+ * @link        https://github.com/JBlond/php-diff
+ */
+interface SubRenderer
```

Comment:
Interfaces names should end with "Interface"

Reply:
Resolved at commit 0ef6def